review/api/dragdrop.api.md

```diff
@@ -34,23 +50,13 @@ export namespace Drag {
         supportedActions?: SupportedActions;
     }
     export function overrideCursor(cursor: string, doc?: Document | ShadowRoot): IDisposable;
+    export type SupportedActions = DropAction | 'copy-link' | 'copy-move' | 'link-move' | 'all';
 }
 
-// @public
-export type DropAction = 'none' | 'copy' | 'link' | 'move';
-
-// @public
-export interface IDragEvent extends MouseEvent {
-    dropAction: DropAction;
-    readonly mimeData: MimeData;
-    readonly proposedAction: DropAction;
-    readonly source: any;
-    readonly supportedActions: SupportedActions;
+// @public @deprecated (undocumented)
+export interface IDragEvent extends Drag.Event {
```

Comment:
As this is deprecating `IDragEvent`, I think we should do the same for `DropAction` and `SupportedActions` as they are part of `IDragEvent` interface.

Otherwise this looks good.